src/vm/eventtrace.cpp

```diff
@@ -1732,7 +1732,7 @@ int BulkTypeEventLogger::LogSingleType(TypeHandle th)
         FireBulkTypeEvent();
     }
     
-    _ASSERTE(m_nBulkTypeValueCount < _countof(m_rgBulkTypeValues));
+    _ASSERTE(m_nBulkTypeValueCount < (int)_countof(m_rgBulkTypeValues));
```

Comment:
Strange, gcc has no issue with line 1730? Is it because of `==` vs `<`?

Reply:
Yeah, I only fixed the warnings that GCC complained about. I checked to see if I can change the type to unsigned but that turned out to be a bigger mess as code returns -1 in some places and index value other places. I left it alone for the moment.